arcade/__init__.py

```diff
@@ -60,10 +60,11 @@ def configure_logging(level: int | None = None):
 import pyglet
 
 # Enable HiDPI support
-if os.environ.get("ARCADE_TEST"):
-    pyglet.options.dpi_scaling = "real"
-else:
-    pyglet.options.dpi_scaling = "stretch"
+# Note: This seems to have changed to 'scale_with_dpi' in pyglet
+# if os.environ.get("ARCADE_TEST"):
+#     pyglet.options['dpi_scaling'] = "real"
+# else:
+#     pyglet.options.dpi_scaling = "stretch"
```

Comment:
I will do a fuller review later, but this has to be fixed as part of the merging of pyglet 2.1.

Reply:
With out that the window is 1/4 of the normal size

Reply:
This section definitely cannot be removed.